src/claims/dto/add-investor-uniqueness.dto.ts

```diff
@@ -12,11 +12,23 @@ export class AddInvestorUniquenessDto extends TransactionBaseDto {
   })
   scope: ScopeDto;
 
+  @ApiProperty({
+    description: 'The CDD ID of the investor',
+  })
```

Comment:
example field is missing. Also all fields should be marked readonly as well along with decorators. Even if there is nothing to add you must add @IsString or somthing like this